app/models/customer.rb

```diff
@@ -0,0 +1,8 @@
+class Customer < ApplicationRecord
+  has_and_belongs_to_many :items
+
+  def add_item(item_id)
```

Comment:
Nice encapsulation of this logic at the model level! This is just .find and << which is all CRUD operations so it's totally fine to leave them in the controller, but I think it cleans things up nicely to have them in their own method!